packages/mbed-greentea/mbed_greentea/mbed_greentea_cli.py

```diff
@@ -686,8 +686,14 @@ def get_parallel_value(value):
             gt_logger.gt_log_warn("entering global resource manager mbed-ls dummy mode!")
             grm_platform_name, grm_module_name, grm_ip_name, grm_port_name = grm_values
             mbeds_list = []
-            for _ in range(parallel_test_exec):
-                mbeds_list.append(mbeds.get_dummy_platform(grm_platform_name))
+            if grm_platform_name is '*':
```

Comment:
I think literal comparisons are frowned upon in Python (my linter always yells at me for doing stuff like this).
```suggestion
            if grm_platform_name == '*':
```